src/server.py

```diff
@@ -372,8 +373,8 @@ def run_jobtypes_query():
 @app.route("/data/create_jobtypes/")
 @json_response
 def run_create_jobtypes_query():
-    # skipping b2g*, android*, mulet*
-    platforms = ['android-2-3-armv7-api9', 'android-4-2-x86', 'android-4-3-armv7-api11', 'osx-10-6', 'osx-10-10', 'windowsxp', 'windows7-32', 'linux32', 'linux64', 'windows8-64']
+    alljobs = jobtype_query()
+    platforms = list(set(job[0] for job in alljobs))
 
```

Comment:
This was already changed and merged in https://github.com/dminor/ouija/pull/77
@jmaher How come it is not present in the repository??